attributes/default.rb

```diff
@@ -4,6 +4,7 @@
 default.loggly.tags = []
 
 default.loggly.log_files = []
+default.loggly.apps = {}
```

Comment:
I'm ok with "apps" if it is idiomatic. I hadn't seen that in practice before :smile: